src/ajax.js

```diff
@@ -254,9 +254,9 @@ function ajaxConvert( s, response, jqXHR, isSuccess ) {
 
 				// Apply converter (if not an equivalence)
 				if ( conv !== true ) {
-
-					// Unless errors are allowed to bubble, catch and return them
-					if ( conv && s[ "throws" ] ) {
+					if ( jQuery.isThenable(response) ) {
```

Comment:
I did it to make it testable and because I was doing the same comparison in multiple places. Don't care too much, will modify.